client/a8c-for-agencies/components/form/hoc/with-error-handling/validators/url.ts

```diff
@@ -0,0 +1,16 @@
+import { translate } from 'i18n-calypso';
+import { isValidUrl } from 'calypso/a8c-for-agencies/sections/partner-directory/utils/tools';
```

Comment:
What about moving the validator tools outside of the partner directory context? Moving that helper function to `components/form` fits better.